src/list.js

```diff
@@ -0,0 +1,78 @@
+import Task from './task';
+
+export default class List {
+  constructor() {
+    this.orderChanged = false;
+    this.tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+  }
+
+  static initialize() {
```

Comment:
I assume this function is called just for deve purposes, and it is not necessary for production.